kafqa.env

```diff
@@ -21,6 +22,11 @@ export CONSUMER_ENABLE_AUTO_COMMIT="true"
 export PROMETHEUS_ENABLED="true"
 export PROMETHEUS_PORT="9090"
 
+export PROTO_PARSER_ENABLED="true"
+export PROTO_PARSER_MESSAGE_NAME="com.test.user.UserLocationLogMessage"
```

Comment:
let's disable this by default, and add the proto in scripts folder (or testdata one) and refer the message and filepath for the same